en/docs/next/ai-workspace/llm-providers/configure-provider.md

```
@@ -87,6 +87,16 @@ The authentication fields vary depending on the provider you selected:
     1. **Upstream URL*** (Required): Enter your Azure AI Foundry endpoint URL.
     2. **API Key*** (Required): Enter your Azure AI Foundry API key.
 
+=== "AWS Bedrock"
+    1. **Upstream URL*** (Required): Enter the Bedrock runtime endpoint for your region, in the form `https://bedrock-runtime.<region>.amazonaws.com` (e.g., `https://bedrock-runtime.us-east-1.amazonaws.com`).
```

markdownlint-cli2 (0.23.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)